x-pack/plugins/ml/public/application/jobs/jobs_list/components/datafeed_modal/constants.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ChartSizeArray } from '@elastic/charts';
+import { i18n } from '@kbn/i18n';
+
+export const CHART_DIRECTION = {
+  FORWARD: 'forward',
+  BACK: 'back',
+} as const;
+export type ChartDirectionType = typeof CHART_DIRECTION[keyof typeof CHART_DIRECTION];
+
+export const CHART_SIZE: ChartSizeArray = ['100%', 300];
```

Comment:
super duper nit: can we add a comment here to clarify if it's 100% width 300px height or the other way around?